tests/ops/api/v1/endpoints/test_health_endpoints.py

```diff
@@ -12,7 +12,7 @@ def test_health(api_client: TestClient) -> None:
         "webserver": "healthy",
         "database": "healthy",
         "version": str(fides.__version__),
-        "workers_enabled": "False",
+        "workers_enabled": False,
```

Comment:
this is the only backend actual change. I didn't see anywhere else that uses `workers_enabled`, so I think it's safe 🤞 I think it makes more sense for it to be a boolean instead of a string "False". we get this for free from using the new `CoreHealthCheck` model

Reply:
That makes sense to me. If the tests are passing and this is the only place it's used then it should be good 🤞